packages/components/src/form-token-field/test/index.js

```diff
@@ -262,7 +262,9 @@ describe( 'FormTokenField', () => {
 
 			// before sending a hover event, we need to wait for
 			// SuggestionList#_scrollingIntoView to become false
-			jest.advanceTimersByTime( 100 );
+			act( () => {
+				jest.advanceTimersByTime( 100 );
+			} );
```

Comment:
Why was this change necessary?

Reply:
Half of the month is gone. Don't remember exact details 😅 . Sorry about that. But yeah test case was failing and after going to test failure. I learned more about act and I was able to fix it. I come to know about `act` from test failure error message as shown in the below image.
<img width="1370" alt="Screenshot 2021-12-16 at 12 24 17 AM" src="https://user-images.githubusercontent.com/21127788/146247883-8933fc89-4fee-4121-bad8-e219075214b5.png">